apis/v1alpha1/forwarding_target_types.go

```diff
@@ -91,3 +91,55 @@ type TargetWeight int32
 //
 // +k8s:deepcopy-gen=false
 type ForwardToTargetObjectReference = ServicesDefaultLocalObjectReference
+
+// GenericForwardToTarget identifies a target object within a known namespace.
+type GenericForwardToTarget struct {
+	// TargetRef is an object reference to forward matched requests to.
+	// The resource may be "services" (omit or use the empty string for the
+	// group), or an implementation may support other resources (for
+	// example, resource "myroutetargets" in group "networking.acme.io").
+	// Omitting or specifying the empty string for both the resource and
+	// group indicates that the resource is "services".  If the referent
```

Comment:
I think we need better wording here to indicate that there are defaults for both group and resource separately.